src/inmanta/agent/forking_executor.py

```diff
@@ -811,7 +811,7 @@ async def execute(
         gid: uuid.UUID,
         resource_details: "inmanta.agent.executor.ResourceDetails",
         reason: str,
-        requires: dict[ResourceIdStr, const.ResourceState],
```

Comment:
We typically use it as the unqualified `Mapping` nowadays. For new code we should follow that approach.